extensions/amp-cache-url/README.md

```diff
@@ -0,0 +1,3 @@
+# Cache URL service
+
+This extension installs a service to convert URLs to cache URLs.
```

Comment:
Please link to https://github.com/ampproject/amp-toolbox/tree/main/packages/cache-url